### ISSUE TITLE
Export range of token serials

### ISSUE THREAD

**Issue by cornelinux** [privacyidea/privacyidea#3949]
**Is your feature request related to a problem? Please describe.**

We would like to export tokens with a range of serial number.

**Describe the solution you'd like**

The best thing would be, to have an export mechanism in the webui.
The tokens would then be marked as "exported", so that these tokens can be deleted later.
The mark could be a key "exported_on" with the timestamp.

**Describe alternatives you've considered**

I could use a regular expression and the token janitor. This is however a bit cumbersome.

**Comment by plettich** [comment]
I don't think it is a good idea to pass the token secrets back through the API.
We deliberately say we can't show the QR-code again for security reasons.

**Comment by cornelinux** [comment]
@plettich Thank you for your thought. 
This is a most important feature since it solves our daily organizational issues.

But it might be sensible to only export it when providing a PGP key.

**Comment by nilsbehlen** [comment]
The general requirement is to match a range of token for an operation, like import, mark, delete etc.
